the TargetPort in svc/harbor-operator is not correct, using helm install

commit: c33624540ccf5ff984193a43e8d963ff45a87b3e

1. make helm-install
2. make sample
3. kubectl describe svc harbor-operator
3. the TargetPort is `webhook/TCP`, which should be `9443/TCP`
![image](https://user-images.githubusercontent.com/188115/94139184-ce22e200-fe9b-11ea-8f40-8a42dc1765be.png)

4. but after `kubectl edit svc harbor-operator`, there will be another cert issue when `make sample`
```
Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook "vharbor.kb.io": Post https://harbor-operator.harbor.svc:443/validate-goharbor-io-v1alpha2-harbor?timeout=30s: x509: certificate signed by unknown authority
Makefile:418: recipe for target 'sample-harbor' failed
```

--- a/charts/harbor-operator/templates/service.yaml
+++ b/charts/harbor-operator/templates/service.yaml
@@ -8,7 +8,7 @@ spec:
   type: {{ .Values.service.type }}
   ports:
     - port: {{ .Values.service.port }}
-      targetPort: webhook
+      targetPort: webhook-server
       protocol: TCP
       name: https
   selector:

